### ISSUE TITLE
CPU view does not show data as zooming out

### ISSUE THREAD

**Issue by tahini** [lttng/lttng-scope#75]
With a fairly big trace (5 minutes of necessary LTTng kernel events for current analyses), the CPU view stops showing any data as I zoom out. See this screencast and observe the CPU view at the bottom, as I zoom out, it ends up totally empty.

https://framapic.org/G7SwYZMTvhKw/ogkv03FFQy5E.gif

**Comment by alexmonthy** [comment]
Interesting, looks like a real bug. Do you get that problem reproducibly, and is it always/only with the CPU view?

**Comment by tahini** [comment]
Yes it is reproducible for the CPU view only. With large 4-5 minutes traces, I get it for all CPUs, but to see the problem faster, I get it from the os-events benchmark trace (http://archive.eclipse.org/tracecompass/test-traces/os-events.tar.gz) for CPU2 only.
